unilog.go

```diff
@@ -126,7 +133,11 @@ func readlines(in io.Reader, bufsize int, shutdown chan struct{}) (<-chan string
 		for err == nil {
 			s, err = r.ReadString('\n')
 			if s != "" {
-				linec <- strings.TrimRight(s, "\n")
+				s = strings.TrimRight(s, "\n")
+				linec <- s
+				if stats != nil {
+					stats.Gauge("unilog.bytes", float64(len(s)), nil, 1.0)
```

Comment:
Yes, this should be a Count, not a gauge.

On that note (and the previous one) - we discussed in Slack what our approach to tagging should be, given that (a) tag cardinality is an issue, (b) we plan on being able to forward counters but can't yet, and (c) hostname may not always be the most meaningful way to identify what we care about (e.g. in a hypothetical containerized world).

So, our plan is to use `--statstag` and (for now) pass in the box type. (In the future, we could get creative with using the service and/or team in the metric name to reduce cardinality and then use the full hostname in the tag, but we don't have a unified naming scheme for metrics (yet) and given a problematic box type, it's easy enough to query splunk to see which host(s) are causing a problem.